client/landing/gutenboarding/onboarding-block/design-selector/page-layout-selector.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import React, { FunctionComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { Template } from '../../stores/verticals-templates/types';
+
+import './page-layout-selector.scss';
+import Card from '../../components/card';
+import CardMedia from '../../components/card/media';
+
+interface Props {
+	templates: Template[];
+}
+
+const PageLayoutSelector: FunctionComponent< Props > = ( { templates } ) => (
+	<div className="page-layout-selector">
+		<div className="page-layout-selector__grid">
+			{ templates.map( template => (
+				<Card key={ template.slug }>
+					<CardMedia>
+						<img alt="FIXME" src={ template.preview } />
```

Comment:
```suggestion
						<img alt={ template.title } src={ template.preview } />
```

is what I used for the Design Selector -- might make sense here too?